SampleUsages/TestSamples/AzureQueueShortRunTests.json

```diff
@@ -0,0 +1,28 @@
+﻿[
+  {
+    "functionName": "Queue-NodeJs-CPUIntensive",
+    "platform": "Azure",
+    "inputPath": "..\\queueitems-static-64.config",
```

Comment:
Yeah I don't think @KrzysiekKnigawka pulled these files in when he made the scenarios. But I think a better solution is to actually provide support for including the messages in the scenario itself. I just want to say `input: ["64"]` here. The relative path is already causing me problems. I filed an issue against @KrzysiekKnigawka: https://github.com/Azure/azure-functions-performance-test/issues/33